gsd-orchestrator/SKILL.md

```
@@ -209,7 +209,7 @@ Available types: `agent_start`, `agent_end`, `tool_execution_start`, `tool_execu
 | `queue` | Queue/reorder milestones |
 | `history` | View execution history |
 | `doctor` | Health check + auto-fix |
-| `knowledge <rule>` | Add persistent project knowledge |
+| `knowledge <rule|pattern|lesson>` | Add persistent project knowledge |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)